src/pages/publication/files_and_license_tab/FileUploader.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import '@uppy/core/dist/style.css';
+import '@uppy/dashboard/dist/style.css';
+
+import Tus from '@uppy/tus';
+import Uppy from '@uppy/core';
+import { Dashboard } from '@uppy/react';
+import { useTranslation } from 'react-i18next';
+
+const uppy = Uppy({
+  autoProceed: true,
+  restrictions: {
+    allowedFileTypes: ['image/*', 'text/*', 'application/*'],
+  },
+}).use(Tus, {
+  endpoint: 'https://master.tus.io/files/',
```

Comment:
samme med denne - bør trekkes ut en plass

Reply:
Denne er bare for testing, når backend er klar må denne fjernes og erstattes med AWS S3-integrasjonen